src/app/components/ResourceHints/index.jsx

```diff
@@ -1,24 +1,32 @@
-import React from 'react';
-import { arrayOf, string } from 'prop-types';
+import React, { Fragment } from 'react';
+import { any, objectOf, string } from 'prop-types';
 
-const ResourceHints = ({ assetOrigins }) => (
-  <>
-    {assetOrigins.map(origin => (
-      <link
-        rel="preconnect"
-        key={origin}
-        href={origin}
-        crossOrigin="anonymous"
-      />
-    ))}
-    {assetOrigins.map(origin => (
-      <link rel="dns-prefetch" key={origin} href={origin} />
-    ))}
-  </>
-);
+const ResourceHints = ({ assetOrigins, service }) => {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const serviceConfig = require(`../../lib/config/services/${service}`);
```

Comment:
Rather than this require, IMO we should move and use https://github.com/bbc/simorgh/blob/latest/src/testHelpers/serviceConfigs.js. As long as we only use it in serverside code like this it, should be fine

Reply:
@EinsteinCarrey We need to move it. We shouldnt rely on a file in `testHelpers` in production code

Reply:
Noted, thanks Drew